docs/source/ontologies_included.md

```diff
@@ -1,38 +1,28 @@
 # Included ontologies
```

Comment:
It could be helpful to shortly list the included ontology with a few words to describe them.

Reply:
Something like a list of bullet points with the names (which should be a link to the section for each ontology) and an accompanying, short sentence?

Reply:
Exactly

Reply:
Done